.env_vars.sample

```diff
@@ -46,5 +46,6 @@ export CONNECT_TEST_ACCOUNT_PASSWORD=
 export CONNECT_TEST_ACCOUNT_USER_ID=
 export CONNECT_ACCOUNT_TEST_URL=
 
-## Connect.Profile
-export CONNECT_PROFILE_URL=
+## Profile
+export CONNECT_PROFILE_URL=http://localhost:21001
+export PROFILE_API_KEY="<api-key-here>"
```

Comment:
I added a script to delete the test user from Connect.Profile's database and we need that to be authorized on the targeted endpoint.